contrib/check-config.sh

```diff
@@ -25,6 +25,10 @@ if ! command -v zgrep > /dev/null 2>&1; then
 	}
 fi
 
+useColor=true
```

Comment:
considering if it would make sense to use `NO_COLOR` (https://no-color.org/) for this; and if not set, set it automatically; something like https://github.com/moby/moby/blob/777e9f271095685543f30df0ff7a12397676f938/hack/dockerfile/install/runc.installer#L12

(commenting from my phone, so apologies for brevity)